ci/cudf_pandas_scripts/pandas-tests/run.sh

```diff
@@ -12,13 +12,15 @@ rapids-logger "PR number: ${RAPIDS_REF_NAME:-"unknown"}"
 
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 
-# Download the cudf and pylibcudf built in the previous step
+# Download the cudf, libcudf, and pylibcudf built in the previous step
 RAPIDS_PY_WHEEL_NAME="cudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+RAPIDS_PY_WHEEL_NAME="libcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp ./dist
 RAPIDS_PY_WHEEL_NAME="pylibcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
```

Comment:
Let's add the cpp/python argument to every invocation of `rapids-download-wheels-from-s3` in this repo. (Fine to do in a later PR if desired)
```suggestion
RAPIDS_PY_WHEEL_NAME="cudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 python ./dist
RAPIDS_PY_WHEEL_NAME="libcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp ./dist
RAPIDS_PY_WHEEL_NAME="pylibcudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 python ./dist
```

Reply:
Sure, I'll do that here. Will accept this in the browse and then push other changes for the other uses. I think it's fine to include in this PR. This PR is adding the first `cpp` wheel to this repo, so the first place where being explicit matters. If I squint hard enough, that makes it in scope 😁

Reply:
handled the rest of these in https://github.com/rapidsai/cudf/pull/15483/commits/5916b9416383000b554ae27fd05d75ca0d71ef6b

They should all now be explicit:

```shell
git grep -E 'download\-wheels'
```